app/assets/stylesheets/layout_components/reveal-modal.scss

```diff
@@ -20,6 +20,25 @@ dialog .close-reveal-modal {
     font-weight: normal;
   }
 
+  &--chat {
+    @extend .reveal-modal;
+    position: fixed;
+    bottom: 0;
+    top: auto !important;
+    right: 20px;
+    z-index: 3;
+    box-shadow: 0 1px 19px 2px rgba(0, 0, 0, 0.25);
+
+    @include breakpoint(small only) {
```

Comment:
I'm noticing the expanded window is breaking a bit on mobile at the moment: 

<img width="398" alt="screen shot 2018-03-08 at 2 03 37 pm" src="https://user-images.githubusercontent.com/27693833/37176435-9a001a10-22d9-11e8-9bd3-badda7033654.png">

Reply:
okay i will fix this